app/views/application_pets/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<h1 class="title">Application Information</h1>
+<ul>
+<li><%= @application.name %></li>
+<li><%= @application.address %></li>
+<li><%= @application.city %></li>
+<li><%= @application.state %></li>
+<li><%= @application.zip %></li>
+<li><%= @application.phone_number %></li>
+<li><%= @application.description %></li>
+</ul>
+<h2>Pets</h2>
+<% @application.pets.each do |pet| %>
+  <section id="pet-<%= pet.id %>">
+    <%= link_to pet.name, "/pets/#{pet.id}"%>
+    <% if pet.application_approved? %>
+      <p><%= pet.name %> is currently pending adoption.</p>
+      <% if !(@application.pending_approval?(pet.id)) %>
```

Comment:
needed a nested if statement to only display revoke link to the application that is approved with that pet